File: b3326rec/app/src/main/java/com/example/b3326rec/MainActivity.java
package com.example.b3326rec;

import androidx.appcompat.app.AppCompatActivity;
import androidx.recyclerview.widget.LinearLayoutManager;
import androidx.recyclerview.widget.RecyclerView;

import android.os.Bundle;

import java.util.ArrayList;

public class MainActivity extends AppCompatActivity {
    ArrayList<ItemLittle> littleItems;
    RecyclerView recyclerView;
    AdapterLittle adapterLittle;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        recyclerView = findViewById(R.id.recycleView);

        littleItems = new ArrayList<>();
        littleItems.add(new ItemLittle("Little 1"));
        littleItems.add(new ItemLittle("Little 2"));
        littleItems.add(new ItemLittle("Little 3"));
        littleItems.add(new ItemLittle("Little 4"));

        recyclerView.setLayoutManager(new LinearLayoutManager(this));
        adapterLittle = new AdapterLittle(littleItems);
        recyclerView.setAdapter(adapterLittle);

    }
}

File: l2052list/app/src/main/java/com/example/l2052list/MainActivity.kt
package com.example.l2052list

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.text.Layout
import android.widget.ArrayAdapter
import android.widget.ListView
import android.widget.TextView
import android.widget.Toast

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        var listView = findViewById<ListView>(R.id.listView)

        var arrayList = ArrayList<String>()
        arrayList.add("Sunday")
        arrayList.add("Monday")
        arrayList.add("Tuesday")
        arrayList.add("")

        val adapter = ArrayAdapter(this,
                android.R.layout.simple_expandable_list_item_1,
                arrayList)
        listView.adapter = adapter
        listView.setOnItemClickListener{parent, view, position, id ->
            Toast.makeText(this,
                    "Pressed item position $position ${arrayList.get(position)}",
                    Toast.LENGTH_LONG).show()
        }
    }
}